build-compose

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+docker-compose --env-file=.env build
```

Comment:
Why do we need to run build through compose? And why do we need the .env?

Reply:
The `.env` file values are filled into the `docker-compose.yml` file during the build step. I tried specifying the env file in the `docker-compose.yml` but when you specify `env_file` in docker-compose, it's only used for running containers, not building, it seems.

Reply:
Oh, and we don't need to run it through compose. It's just an option. It's a convenient wrapper on the Docker build process, and I have been using it while testing out the docker-compose setup.